### PR TITLE
feat:implementeed fighter page

### PR DESCRIPTION
**Pull request by albin1616**
implemented fighter

### CODE REVIEW

File: fight.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function fight(a){
```
Comment (Rajharsh2) [inline fight.js:1]: - Please give task specific function name.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: fight.js (hunk 1 above)
Comment (manvithalva12) [inline fight.js:1]: please pass both a and b parameters for the function

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

File: fight.js (hunk 1 above)
Comment (albin1616) [reply fight.js]: resolved

Comment (albin1616) [reply fight.js]: resolved

**Review by manvithalva12 — APPROVED** [review]
LGTM

File: fight.js (hunk 1 above)
Comment (Rajharsh2) [reply fight.js]: lgtm

**Review by Rajharsh2 — APPROVED** [review]

**Pull request merged**